app/connectors/MinimalDetailsConnector.scala

```diff
@@ -56,6 +56,7 @@ class MinimalDetailsConnectorImpl @Inject() (appConfig: FrontendAppConfig, http:
     http
       .get(url"$url")
       .setHeader(idType -> idValue, "loggedInAsPsa" -> loggedInAsPsa.toString)
+      .transform(_.withRequestTimeout(appConfig.ifsTimeout))
```

Comment:
Unless there is a specific reason, I don't think it is needed here, as this is a different service (not IF).